internal/server/singleprocess/state/app_operation.go

```diff
@@ -50,6 +61,9 @@ type appOperation struct {
 	seq map[string]map[string]*uint64
 }
 
+// By default, we allow 10000 records
+const DefaultMaximumIndexedRecords = 10000
```

Comment:
I don't think we use this anywhere.

If we do, I don't mean to get too wordy, but I think we should try to do like `DefaultAppOperationMaxIndexedRecords` (ugh such Java), but just to be clear that this is for app ops only?

If we intend this to be for _anything_ this is fine but it should probably go in a diff file.

Reply:
I just removed it for now since we're not using it.